### PR TITLE
Fix typo’s and punctuation improvements

### PR DESCRIPTION
**Pull request by rensverschuren**

### CODE REVIEW

File: config/locales/nl.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ nl:
   sign_in: Inloggen
   locale_name: "Nederlands"
   sign_up: Aanmelden
-  footer_about: "RubyGems.org is de gem hosting service van de Ruby community. Publiceer and installeer je gems direct. Gebruik de API om meer informatie van beschikbare gems te vinden. Wordt een deelnemer en verbeter de site met jouw aanpassingen."
```
Comment (arthurnn) [inline config/locales/nl.yml]: I dunno dutch, but I think this was correct before. see https://translate.google.com/#nl/pt/Word%20een%20deelnemer%20en%20verbeter

Comment (jvanbaarsen) [inline config/locales/nl.yml]: @arthurnn The fix is correct. (Weird dutch language rules :D I made a mistake there ;) )

Comment (arthurnn) [inline config/locales/nl.yml]: cool

**Comment by arthurnn** [comment]
thanks

**Pull request merged**